maintainers/maintainer-list.nix

```
@@ -20833,13 +20833,18 @@
     githubId = 104558;
     name = "Benjamin Saunders";
   };
+  ralleka = {
+    name = "RalleKa";
+    github = "RalleKa";
+    githubId = 46349331;
+  };
   ramblurr = {
     name = "Casey Link";
     email = "nix@caseylink.com";
     github = "Ramblurr";
     githubId = 14830;
     keys = [ { fingerprint = "978C 4D08 058B A26E B97C  B518 2078 2DBC ACFA ACDA"; } ];
-  };
+ };
```

```suggestion
  };
```
Should not've done that. Run `nix fmt` or `nix-shell --run treefmt` and squash the fix to the `maintainers: ...` commit

---

Yup, treefmt action already complains about that line